src/tasks/qa.py

```diff
@@ -17,6 +17,27 @@
 from .registry import register_task
 
 
+def _get_f1(x, y):
```

Comment:
Why do we need this? Add a header comment.

Reply:
ReCoRD's metrics involve computing token-level F1 between a prediction and gold answer. This is my version, but I'm adding in the function from the official eval script.